Add package requirements for running openqa-label-known-issues (#336)
os-autoinst/scripts: README.md

## How to use

Checkout the individual scripts and either call them manually or automatically, e.g. in CI jobs

> Find all the dependencies which are required in [dependencies.yaml](https://github.com/os-autoinst/scripts/blob/master/dependencies.yaml]. You get all of them installed with one package.

Reviewer: Please wrap at 80 characters

Author: done https://github.com/os-autoinst/scripts/commit/9d9b52d5870aa14ff53f17b7e14d8f19b5f401b1

> Find all the dependencies which are required in [dependencies.yaml](https://github.com/os-autoinst/scripts/blob/master/dependencies.yaml]. You get all of them installed with one package.
> 
> ```bash
> sudo zypper in os-autoinst-scripts-deps
> ```

Reviewer 2: I would keep it short. There's no reason to explain how to use `zypper` here.

```suggestion
Required dependencies are listed in
[dependencies.yaml](https://github.com/os-autoinst/scripts/blob/master/dependencies.yaml].
You can install all dependencies under openSUSE via the `os-autoinst-scripts-deps` package.
```

Author: it is not to explain zypper. I would like to keep it. it is easier when you look up to get started and easier when you forget that particular thing, as opposed to read through the text. I hope it is not a problem.
